src/lib/Zikula/Core/AbstractTheme.php

```diff
@@ -77,6 +77,51 @@ public function generateThemedBlock(array $block)
     }
 
     /**
+     * load the themevars into the themeEngine global vars
+     */
+    public function setThemeVars()
```

Comment:
I don't like the name too much - *setting* a *var* in Zikula context always meant a function like `setVar($name, $value)`. I'd rather change this to `initThemeVars` or `setThemeVarsInContainer` or something.